[DOCS] 7.13.0 release notes (#99978)
elastic/kibana: docs/CHANGELOG.asciidoc

// * <<release-notes-7.0.0-alpha1>>

--

[[release-notes-7.13.0]]

> == {kib} 7.13.0

Reviewer: @jen-huang @ph Noting here that the Kibana release notes have quite a few items related to Fleet and Elastic Agent. Do we duplicate these points in the release notes we provide in the Fleet User Guide, or point to these release notes? I do not like to send users on easter egg hunts, but duplication can also be problematic.

coming::[7.13.0]

For information about the {kib} 7.13.0 release, review the following information.

[float]
[[breaking-changes-7.13.0]]
=== Breaking changes

Breaking changes can prevent your application from optimal operation and performance.
Before you upgrade to 7.13.0, review the breaking changes, then mitigate the impact to your application.

// tag::notable-breaking-changes[]

[discrete]
[[breaking-97206]]
.Remove Elastic Agent routes and related services
[%collapsible]
====
*Details* +

> You are now unable to run and manage Elastic Agent with {kib}. For more information, refer to {kibana-pull}97206[#97206].

Reviewer: This description is potentially misleading because the Fleet UI lives within Kibana. The main point here is that Elastic Agents will no longer communicate with Kibana to enroll agents, get agent policies, collect status info, and so on. Instead, the agents will use Fleet Server as a control plane. It might be worth pointing to the Fleet Server docs.

The description here makes it sound like the user is _losing_ something when really they are actually gaining scalability. :-) 

@mostlyjason Can you provide more guidance on how to phrase this?

* Expose session invalidation API {kibana-pull}92376[#92376]
Sharing::
* Allows editing broken visualizations caused by runtime fields changes {kibana-pull}94798[#94798]
Uptime::
* Synthetics - add transfer size, resource size, and status to waterfall flyout {kibana-pull}97042[#97042]
* Preconfigured alert history index connector {kibana-pull}94909[#94909]
* Adds Custom Fleet Integration UI {kibana-pull}91584[#91584]
* Support agent data streams {kibana-pull}91469[#91469]
* Synthetic check steps list view {kibana-pull}90978[#90978]
* Service overview: Introduce time-series comparison {kibana-pull}88665[#88665]
* Search made easy {kibana-pull}88581[#88581]

[float]
[[fixes-v7.13.0]]
=== Bug Fixes
Alerting::
* Fixing Elasticsearch query rule to allow matching on 0 documents {kibana-pull}97735[#97735]
* Create task to cleanup action execution failures {kibana-pull}96971[#96971]
* Skip rendering empty add action variables button as disabled {kibana-pull}96342[#96342]
* Ensures ES Query uses the correct sort value to avoid duplicate detections {kibana-pull}94349[#94349]
* Deprecates reloadAlerts and replaces with onSave in Alert creation and editing {kibana-pull}91997[#91997]
APM::

> * Transaction type is not persisted from service inventory {kibana-pull}99407[#99407]
> * Service overview page fetches data with wrong transaction type {kibana-pull}98657[#98657]
> * Transaction duration histogram buckets without samples are clickable {kibana-pull}98540[#98540]
> * Bug: Fix header controls (type, search, comparison, and date picker) responsive breakpoints {kibana-pull}98179[#98179]
> * Home page is showing incorrect value of APM throughput (tpm) {kibana-pull}95991[#95991]
> * Fixes spaces issue for static index patterns {kibana-pull}95799[#95799]
> * The Observability Getting Started page infinitely redirects, breaking browser's back button {kibana-pull}95083[#95083]

Reviewer 2: ```suggestion
* Persist transaction type when navigating from service inventory {kibana-pull}99407[#99407]
* Fetch data from the service overview with correct transaction type {kibana-pull}98657[#98657]
* Transaction duration histogram buckets without samples are clickable {kibana-pull}98540[#98540]
* Fix responsive breakpoints in header controls {kibana-pull}98179[#98179]
* Fix incorrect APM throughput (tpm) on home page {kibana-pull}95991[#95991]
* Fix spaces issue for static index patterns {kibana-pull}95799[#95799]
* Prevent infinite redirects on Observability getting started page {kibana-pull}95083[#95083]
```

Design::
* A11y improvements, focus glitches fixes, unskip tests {kibana-pull}94148[#94148]
Discover::
* Exclude metafields from fields request {kibana-pull}99443[#99443]
* Request docvalue fields from fields {kibana-pull}98874[#98874]
* Fixes redirect from Dashboard {kibana-pull}97098[#97098]
* Sort option: Introduce format parameter {kibana-pull}96774[#96774]
* Remove redundant execution of onRequestStart when fetching data {kibana-pull}94093[#94093]
* The `discover:aggs:terms:size` advanced setting got removed since it was no longer used {kibana-pull}93437[#93437]
* Close inspector when switching app {kibana-pull}92994[#92994]
Elastic Security::
For the Elastic Security 7.13.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
Fleet::
* Correctly parse falsy YAML fields in agent policy integrations {kibana-pull}95966[#95966]
* Match telemetry key names to UI agent states {kibana-pull}95567[#95567]
* Ignore inactive agents when removing a policy {kibana-pull}94311[#94311]
* Correctly track install status of an integration {kibana-pull}93464[#93464]

> * Fixes default integration name when adding 10+ names {kibana-pull}93278[#93278]

Reviewer: ```suggestion
* Fix default integration name when adding 10+ names {kibana-pull}93278[#93278]
```

Kibana Home & Add Data::
* Update Cloud plugin to handle new config in kibana.yml {kibana-pull}95569[#95569]
Lens & Visualizations::
* Visualize runtime fields in *TSVB* {kibana-pull}95772[#95772]
* Enables url drilldowns for range selection in *TSVB* {kibana-pull}95296[#95296]
* Enable `dual mode`, support index patterns and strings in *TSVB* {kibana-pull}92812[#92812]
Machine Learning::
* Anomaly detection rule lookback interval improvements {kibana-pull}97370[#97370]
* Adds network ML module with four ML jobs for ECS network data {kibana-pull}96480[#96480]
* Adds runtime support for anomaly charts & add composite validations {kibana-pull}96348[#96348]
* Data frame analytics: Adds support for runtime fields {kibana-pull}95734[#95734]
* Adds Anomaly Explorer charts embeddable {kibana-pull}94396[#94396]
* Data frame analytics creation wizard: Add validation step {kibana-pull}93478[#93478]
* Adding support for saved object based ML modules {kibana-pull}92855[#92855]
* Adds search time runtime support for index based Data Visualizer {kibana-pull}95252[#95252]
Metrics::

> * Observability Overview Host Summary {kibana-pull}90879[#90879]

Reviewer 3: ```suggestion
* Enhanced metrics widget on Observability overview page {kibana-pull}90879[#90879]
```

* Integrates the color service in *TSVB* {kibana-pull}93749[#93749]
* Change the default mode from last value to entire timerange in *TSVB* {kibana-pull}93608[#93608]
* Enable `dual mode`, support index patterns and strings in *TSVB* {kibana-pull}92812[#92812]
* Implement filtered metric in *Lens* {kibana-pull}92589[#92589]
* Show an indicator when using Last Value mode in *TSVB* {kibana-pull}91977[#91977]
* Adds the runtime field editor to *Lens* {kibana-pull}91882[#91882]
* Introduces new chart switcher in *Lens* {kibana-pull}91844[#91844]
* Control round and decimal places in Gauge Visualization when using aggregate functions like average {kibana-pull}91293[#91293]
* Integrates legend color picker with the eui palette {kibana-pull}90589[#90589]
* Transpose columns in *Lens* {kibana-pull}89748[#89748]
* Support for number, date and IP range data types {kibana-pull}76971[#76971]
Logs::
* Support runtime fields in the log threshold alert {kibana-pull}97603[#97603]
* Display and query runtime fields from KIPs in the log stream and entry flyout {kibana-pull}97467[#97467]
* Adds error.stack_trace to the default log formatter {kibana-pull}94906[#94906]

> * Support Kibana index patterns in the Logs UI settings UI {kibana-pull}94849[#94849]

Reviewer 3: ```suggestion
* Support Kibana index patterns in the Logs app settings {kibana-pull}94849[#94849]
```

* Support max primary shard size rollover field {kibana-pull}96545[#96545]
* Move other bucket into Search Source {kibana-pull}96384[#96384]
* Added readonly toggle in cold phase {kibana-pull}96036[#96036]
* Adds "Include aliases" toggle to the Restore Snapshot Wizard {kibana-pull}95882[#95882]
* Display time labels in search session popover {kibana-pull}95082[#95082]
* Make session name editable {kibana-pull}94610[#94610]
* Cloud deployment form when adding new cluster {kibana-pull}94450[#94450]
* Adds multitable support to Inspector {kibana-pull}94077[#94077]
* Adds support for frozen phase {kibana-pull}93068[#93068]
* Index pattern field editor {kibana-pull}88995[#88995]
Maps::
* Show layer name in tooltip {kibana-pull}97175[#97175]
* Adds visibility toggle and fit to bounds buttons to layer quick actions menu {kibana-pull}97013[#97013]
* Enable filtering with spatial relationships on geo_point fields {kibana-pull}96849[#96849]
* Enable distance filtering on geo_shape {kibana-pull}96832[#96832]
* Adds cache control to mvt endpoints {kibana-pull}94861[#94861]
* Do not track total hits for elasticsearch search requests {kibana-pull}91754[#91754]
Metrics::

> * Increase groupBy pagination size on Metric Threshold alerts {kibana-pull}99683[#99683]

Reviewer 3: ```suggestion
* Increase the groupBy pagination size on metric threshold alerts {kibana-pull}99683[#99683]
```

> * Adds anomalies tab to enhanced node details {kibana-pull}96967[#96967]

Reviewer 3: ```suggestion
* Adds anomalies tab to enhanced host details overlay {kibana-pull}96967[#96967]
```

> * Handle ES query errors in snapshot endpoint, improve UI error messaging {kibana-pull}96871[#96871]

Reviewer 3: ```suggestion
* Improved UI error messaging for handling ES query errors in a snapshot endpoint {kibana-pull}96871[#96871]
```

> * Adds Log Rate to the metrics tab {kibana-pull}96596[#96596]

Reviewer 3: ```suggestion
* Adds log rate tab to enhanced host details overlay {kibana-pull}96596[#96596]
```

* Change composite.size of snapshot ES query to improve speed {kibana-pull}95994[#95994]
* Change top_hits aggregation to top_metrics for snapshot api {kibana-pull}95644[#95644]
* Adds preview charts to Inventory alerts {kibana-pull}91658[#91658]

> * Observability Overview Host Summary {kibana-pull}90879[#90879]

Reviewer 3: ```suggestion
* Enhanced metrics widget on Observability overview page {kibana-pull}90879[#90879]
```

> * Adds anomalies table to flyout {kibana-pull}90305[#90305]

Reviewer 3: ```suggestion
* Adds anomalies table to Metrics app {kibana-pull}90305[#90305]
```
